### ISSUE TITLE
Last update broke webpack sass styles importing

### ISSUE THREAD

**Issue by steida** [benmosher/eslint-plugin-import#192]
```
src/browser/app/App.react.js
  1:8  error  Parse errors in imported module './App.scss': Line 3: Unexpected token (3:1)  import/namespace
```

I am not sure what should I do on my side. This should work: https://github.com/este/este/blob/master/src/browser/app/App.react.js#L1

**Comment by benmosher** [comment]
Do you have an import/ignore pattern set up for .scss files? https://github.com/benmosher/eslint-plugin-import/blob/master/README.md#importignore

On Feb 25, 2016, at 19:13, Daniel Steigerwald <notifications@github.com<mailto:notifications@github.com>> wrote:


src/browser/app/App.react.js
  1:8  error  Parse errors in imported module './App.scss': Line 3: Unexpected token (3:1)  import/namespace


I am not sure what should I do on my side. This should work: https://github.com/este/este/blob/master/src/browser/app/App.react.js#L1

-

**Comment by benmosher** [comment]
Actually, nvm. I'll fix it on my end. Shouldn't try to parse if there are no specifiers.

**Comment by steida** [comment]
Awesome! Thank you!

**Comment by benmosher** [comment]
Published! :sweat_smile: v1.0.2.

FWIW, `no-deprecated` will still have this issue, but 

```yaml
# .eslintrc.yml
---
settings:
  import/ignore: 
    - \.scss$
```

would also fix it. That [setting](https://github.com/benmosher/eslint-plugin-import/blob/master/README.md#importignore) instructs the dependency parser to ignore errors.
